src/libraries/Common/src/System/Runtime/InteropServices/GeneratedMarshallingAttribute.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.InteropServices
```

Comment:
Might be worth pointing at the runtimelab repo so people know what is consuming these types. I would also do the same with `GeneratedDllImportAttribute`.